ProjectLighthouse.Localization/Moderation.resx

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<root>
+    <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+        <xsd:element name="root" msdata:IsDataSet="true">
+            
+        </xsd:element>
+    </xsd:schema>
+    <resheader name="resmimetype">
+        <value>text/microsoft-resx</value>
+    </resheader>
+    <resheader name="version">
+        <value>1.3</value>
+    </resheader>
+    <resheader name="reader">
+        <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <resheader name="writer">
+        <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <data name="banned_heading" xml:space="preserve">
+        <value>Account Banned</value>
+    </data>
+    <data name="banned_explain" xml:space="preserve">
+        <value>Your account has been banned due to inappropriate, offensive, or violent behavior.</value>
```

Comment:
Probably better to say something more vague like `You have been banned for violating the rules in our terms of service.`

Reply:
This is to keep consistent with what is displayed publicly on banned users profiles, but I'll take your word for it.

Reply:
You could change the public-facing message as well to keep it consistent, I feel it's too verbose anyway